manifests/server/export.pp

```diff
@@ -55,6 +58,15 @@
 #     nfs_v4_export_root_clients => '1.2.3.4/24(rw,fsid=root,insecure,no_subtree_check,async,no_root_squash)',
 #   }
 #
+# nfs::server::export { '/srv/nfs_exported/directory':
+#   clients => '1.2.3.4/24(rw,insecure,no_subtree_check,async,no_root_squash) 5.6.7.8/24(ro)',
+#   share => 'share_name_on_nfs_server',
+# }
+#
+# Links
+#
+# * {Puppet Docs: Using Parameterized Classes}[http://j.mp/nVpyWY]
```

Comment:
can you use the original link? the shortner is dead. puppet-strings also has a real annotation:

`# @see my awesome description https://blablabla.de`

Reply:
edit: maybe first the link, then the description :thinking: